adafruit_requests.py

```diff
@@ -358,7 +358,10 @@ def _parse_headers(self) -> None:
                     self._remaining = int(content)
                 if title == "transfer-encoding":
                     self._chunked = content.strip().lower() == "chunked"
-                self._headers[title] = content
+                if title == "set-cookie" and title in self._headers:
+                    self._headers[title] = self._headers[title] + ", " + content
```

Comment:
This could be simplified to:

```python
self._headers[title] += ", " + content
```

Reply:
Thanks for the feedback. I've committed this to my branch and it's showing up in the pull request now.